src/helpers.js

```diff
@@ -0,0 +1,7 @@
+import cockpit from "cockpit";
+
+const _ = cockpit.gettext;
+
+export const caKeys = {
+    SelfSign: _("Self-signed")
```

Comment:
I am not sure sure if this will be translated as it will be imported before cockpit.lang is set.

Reply:
Oh yes, I knew that. Will adjust.